civictechprojects/static/css/partials/_PositionList.scss

```diff
@@ -0,0 +1,51 @@
+.PositionList-entry {
+  background-color: $color-background-light;
+  box-shadow: 0 3px 5px rgba(0,0,0,0.2);
+  padding: 5px;
+  margin: 15px;
+  max-width: 600px;
+  display: flex;
+  justify-content: space-between;
+  cursor: grab;
+
+  i {
+    cursor: pointer;
+    margin: auto 0;
+  }
+  i:hover {
+    color:$color-orange;
+  }
+}
+
+.PositionList-entry.dim {
+  color:$color-grey-disabled;
+}
+
+.PositionList-entry.grabbed {
+  cursor: grabbing;
+  background-color: $color-text-grey;
```

Comment:
Rather than the dark text grey for a background, $color-grey-medium may be better here for now - it's more readable when you have the item grabbed. I'm sure we'll be redoing it with a different hover color when the design system gets there but this is good enough for now I think.

Reply:
Yep, that color looks much better.